docs/design/quick_filters.rst

```diff
@@ -0,0 +1,121 @@
+Quick filters for searches
+==========================
+
+Quick filters provide an efficient way to search using existing search commands. This documentation outlines how to create and implement new quick filters for Mautic searches.
+
+Implementation overview
+-----------------------
+
+Quick filters are implemented using a combination of JavaScript and Twig templates. The process involves three main components:
+
+1. JavaScript function for applying the filter
+2. Twig template for rendering filter buttons
+3. Array of PHP code for defining filter options
+
+JavaScript functionality
```

Comment:
```suggestion
JavaScript features available
```

Reply:
it could be strange to talk about features (things that generate a possible action on the UI) when in fact we're talking about the gears from the system

what do you think?